indra/newview/llagentcamera.cpp

```diff
@@ -2204,10 +2204,6 @@ void LLAgentCamera::changeCameraToMouselook(bool animate)
         mCameraMode = CAMERA_MODE_MOUSELOOK;
         const U32 old_flags = gAgent.getControlFlags();
         gAgent.setControlFlags(AGENT_CONTROL_MOUSELOOK);
-        if (old_flags != gAgent.getControlFlags())
-        {
-            gAgent.setFlagsDirty();
-        }
```

Comment:
The "dirty flag" info in `LLAgent `has been removed.  It is no longer helpful.  Instead, the `send_agent_update()` logic checks the current value against the last one sent to decide if something changed.